app/models/bookmark.rb

```diff
@@ -124,10 +124,11 @@ def invalidate_bookmark_count
     end
   end
 
-  # We index the bookmark count, so if it should change, update the pseud
-  def update_pseud_index
+  # We index the bookmark count for pseuds and collections, so update those if they should change
+  def update_pseud_and_collection_index
```

Comment:
Question for AD&T folks: are we good with combining these two concepts here, or do we want to keep them separate like in the Work model?

Reply:
I think we could combine the callbacks in Work, since the indexing triggers are exactly the same.